portfolio/src/main/webapp/contact.html

```diff
@@ -16,35 +18,40 @@
 		<div class="nav-right-side">
 			<a href="https://www.linkedin.com/in/roland-naijuka" target="_blank"
 				class="nav-button navbar-item"><img src="/images/linkedin.png" alt="My LinkedIn"></a>
-			<a href="mailto:naijuka.roland@outlook.com" target="_blank"
-				class="nav-button navbar-item"><img src="/images/mail.png" alt="Email me"></a>
+      <a href="mailto:naijuka.roland@outlook.com" target="_blank"
+        class="nav-button navbar-item"><img src="/images/mail.png" alt="Email me"></a>
 		</div>
 	</nav>
 
 	<div class="contact-me">
 		<h1>Contact Me!</h1>
-		<form action="/data" method="POST">
+		<form id="comment-form" method="POST" enctype="multipart/form-data">
 			<div class="form-content">
 				<label for="name">Name</label>
 				<input type="text" name="name" id="name" placeholder="optional"><br>
-            </div>
-			<div class="form-content">
-                <label for="comment">Comments</label>
-                <textarea id="comment" name="comment" rows="7" cols="30" placeholder="Type your comments here!" required></textarea><br>
-            </div>
-            <button role="submit" name="commentButton">Submit</button>
+      </div>
+      <div class="form-content">
+        <label for="name">Upload an image</label>
+        <input type="file" name="image"><br>
+      </div>
+      <div class="form-content">
+        <label for="comment">Comments</label>
+        <textarea id="comment" name="comment" rows="7" cols="30" placeholder="Type your comments here!" required></textarea><br>
```

Comment:
nit: margin's and padding allow for better control over spacing rather than `<br>`

Reply:
I will look into that, thanks Lev